.github/stale.yml

```diff
@@ -0,0 +1,22 @@
+# See https://github.com/apps/stale
+
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 60
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 14
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - "need: discussion"
```

Comment:
Perhaps keep the `cleanup` label too? As that's an internal label collaborators may add to tickets

Reply:
good call